src/layout/components/HomePage/style.less

```diff
@@ -38,6 +38,24 @@
     font-weight: 700;
     margin: 40px 0;
   }
+
+  .ant-dropdown-link {
+    color: black;
+    cursor: pointer;
+    font-size: @general-body;
+    font-weight: bold;
+    padding-bottom: 1rem;
+    width: fit-content;
+
+    svg {
+      margin-left: 5px;
+      vertical-align: middle;
+    }
+  }
+
+  .ant-dropdown-menu-item {
+    width: fit-content !important;
```

Comment:
General question: did we ever decide whether using `!important` in the portal stylesheets was allowed or not? I don't recall.